server/controllers/list.js

```diff
@@ -111,18 +113,31 @@ const getArchivedListsMetaData = (req, resp) => {
   );
 };
 
+const checkIfCurrentUserVoted = (item, userId) =>
+  item.voterIds.indexOf(userId) > -1;
+
+const itemToSend = (item, userId) => {
+  const { voterIds, ...rest } = item.toObject();
+  return {
+    ...rest,
+    didCurrentUserVoted: checkIfCurrentUserVoted(item, userId)
+  };
+};
+
 const addItemToList = (req, resp) => {
   const {
-    item: { name, isOrdered, authorName, authorId, voterIds },
+    item: { name, isOrdered, authorName, authorId },
```

Comment:
It's not a main goal but maybe isOrdered should be removed here also (and from InputBar at the frontend). We could set it's default value to false in the model... there is no possibility to set this value during item cration.